packages/vitest/src/node/cli.ts

```diff
@@ -151,6 +153,9 @@ function normalizeCliOptions(argv: CliOptions): CliOptions {
   else
     delete argv.dir
 
+  if (argv.exclude)
+    argv.exclude = [...defaultExclude, ...toArray(argv.exclude)]
```

Comment:
I don't think we should do that. We probably want to extend user's `exclude` pattern, not the default one. Maybe we can just move it into another field (`extraExclude`/`cliExclude`) and spread it in the `vitest/src/config.ts` file?

Reply:
Solved by ae05f63